tests/data/serialization/ndjson/test_text.py

```diff
@@ -1,20 +1,37 @@
 import json
+from labelbox.data.annotation_types.annotation import ClassificationAnnotation
+from labelbox.data.annotation_types.classification.classification import ClassificationAnswer, Radio, Text
+from labelbox.data.annotation_types.data.text import TextData
+from labelbox.data.annotation_types.label import Label
 
 from labelbox.data.serialization.ndjson.converter import NDJsonConverter
 
 
-def test_text():
-    with open('tests/data/assets/ndjson/text_import.json', 'r') as file:
```

Comment:
Should the JSON files be removed or are they still needed?

Reply:
@whistler remove those files, great catch!